ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingQueue.java

```diff
@@ -171,6 +172,7 @@ public OptimizingTask pollTask(String authToken, int threadId) {
     if (task != null) {
       safelySchedule(task, new OptimizingThread(authToken, threadId));
       executingTaskMap.putIfAbsent(task.getTaskId(), task);
+      loadTableForTaskInput(task);
```

Comment:
It's possible that we have to load the table every time polling a task, it may become a bottleneck since there may be plenty of tasks.

Reply:
The checking is inside the method.

Reply:
As far as I understand, the loading table operation happens only for tasks recovered from the database after restarting AMS, which is accepted.

Reply:
OK, got it.